### PR TITLE
4 build ux/UI for list events

### PR DESCRIPTION
**Pull request by Jan-OkeMettendorf**
First implementation of the graphical layout of the app completed. Extension of the page for displaying groups is done. Groups are displayed via a JSON file. Implementation of backend not yet done and will be mapped when UserStory 5 is completed. Entry into detail page (UserStory 5) of a group already prepared.

### CODE REVIEW

**Review by macben888 — APPROVED** [review]

**Review by macben888 — CHANGES_REQUESTED** [review]
change the jdk version in your *.yml file to match the project's jdk version

File: frontend/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    );
+}
+
+export default Header;
+
+const StyledHeader = styled.header`
+  display:grid;
+  background-color: black;
+  width: 100vw;
+  height: 30px;
```
Comment (christophersiem) [inline frontend/src/components/Header.js:29]: check if you really need to set those values

File: frontend/src/components/groups.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "participationFee": "5 EUR",
+    "region": "Hamburg",
+    "location" : "Musterstraße 1",
+    "member": [
+      {
+        "firstName": "Mona",
+        "LastName": "Mettendorf",
+        "e-mail": "asd@asd.de",
+        "e-mail-hidden": true,
+        "mobilNumber": null,
```
Comment (christophersiem) [inline frontend/src/components/groups.json:19]:
```suggestion
        "mobileNumber": null,
```

File: frontend/src/components/groups.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        "mobilNumber-hidden": true,
```
Comment (christophersiem) [inline frontend/src/components/groups.json:20]:
```suggestion
        "mobileNumber-hidden": true,
```

File: frontend/src/components/groups.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "bringsBall": true,
+        "bringsBeer": true
+      },
+      {
+        "firstName": "Jan-Oke",
+        "LastName": "Mettendorf",
```
Comment (christophersiem) [inline frontend/src/components/groups.json:26]:
```suggestion
        "lastName": "Mettendorf",
```

File: frontend/src/components/groups.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
+
+[
+  {
+    "id": "1",
+    "title": "Group1",
+    "sport": "football",
+    "CountOfMember": 12,
```
Comment (christophersiem) [inline frontend/src/components/groups.json:7]:
```suggestion
    "countOfMember": 12,
```

File: frontend/src/components/Sidebar.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+const StyledLi = styled.li`
+  display: flex;
+  color:white;
+  font-weight: bold;
+  justify-content: center;
+  align-items: center;
+  height: 50px;
+`
+
+const StyledNavLink = styled(NavLink)`
```
Comment (christophersiem) [inline frontend/src/components/Sidebar.js:52]: i think you don't need to set `display : flex` repeatedly, pls check if you can leave it away

File: frontend/src/App.js [hunk 7]
```diff
@@ -1,24 +1,26 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div className="App">
+          <RoutedPages/>
+        <GlobalStyle/>
```
Comment (christophersiem) [inline frontend/src/App.js:22]: Good idea to use a `GlobalStyle`, but seems like you don't use it yet :)

**Review by christophersiem — APPROVED** [review]
Like the design! Left a few comments 🙌🏽

File: frontend/src/components/NavBar.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+
+function NavBar() {
+
+    const [showSidebar, setShowSidebar] = useState(true);
+
+
+    const handleSidebar = (event) => {
+        setShowSidebar(event)
+    }
```
[the remark this answers is not in the document]
Comment (Jan-OkeMettendorf) [reply frontend/src/components/NavBar.js:13]: done! Thank you! :)

File: frontend/src/components/Groups-data.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+            </StyledInformationLeft>
+            <StyledInformationRight>
+                Gebühr: {group.participationFee}
+            </StyledInformationRight>
+            <StyledInformationRight>
+                <Icons.RiArrowLeftSFill/>
+                {group.member.length} / {group.CountOfMember}
+                <Icons.RiArrowRightSFill/>
+            </StyledInformationRight>
+        </>
```
[the remark this answers is not in the document]
Comment (Jan-OkeMettendorf) [reply frontend/src/components/Groups-data.js:29]: done! thank you! :)

File: frontend/src/components/Sidebar.js (hunk 6 above)
Comment (Jan-OkeMettendorf) [reply frontend/src/components/Sidebar.js:52]: I think I need to set display : flex to get the right gap between the icon and the corresponding texts

**Pull request merged**